src/components/blogPostPreview.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import styled from 'styled-components'
+import PageLink from './page-link'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+
+const BlogPostPreview = ({ post }) => {
+  console.log(post)
+  return (
+    <>
```

Comment:
I don't think there is any need for a fragment here? docs: https://reactjs.org/docs/fragments.html